.aicoe-ci.yaml

```diff
@@ -1,6 +1,5 @@
 check:
-  - thoth-precommit
   - thoth-build
 build:
-  base-image: "quay.io/thoth-station/s2i-minimal-notebook:v0.0.6"
+  base-image: "quay.io/thoth-station/s2i-minimal-py38-notebook:v0.3.0"
```

Comment:
wasn't pytorch's base image the minimal CUDA one? or is this something else?

Reply:
This is just for normal build testing, this doesnt go into the final image.
It is just used for the build-check in the github PR, as you can see below.
and yes, CUDA minimal image used based in the build pipeline finally.